hadoop-ozone/dist/src/main/compose/upgrade/test.sh

```diff
@@ -32,12 +32,7 @@ run_test_scripts ${tests} || RESULT=$?
 RESULT_DIR="$ALL_RESULT_DIR" create_results_dir
 
 # Upgrade tests to be run.
-# Run all upgrades even if one fails.
-# Any failure will save a failing return code to $RESULT.
-set +e
-run_test manual-upgrade 0.5.0 1.1.0
-run_test non-rolling-upgrade 1.0.0 1.1.0
```

Comment:
I didn't see these tests (and old version-specific logic) being removed mentioned in the PR.  I'm fine with it, but maybe the README should be updated accordingly.  Specifically: _... testing upgrades from any previous version to another previous version_ is no longer true (if I understand it correctly).

Reply:
1.0.0 -> 1.1.0 did not require any manual intervention. So the upgrade framework supports upgrading from 1.0.0 (GA) to any later version.